cdcv2/owner/owner.go

```diff
@@ -0,0 +1,318 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package owner
+
+import (
+	"context"
+	"database/sql"
+	"io"
+	"sync"
+	"sync/atomic"
+	"time"
+
+	"github.com/pingcap/log"
+	"github.com/pingcap/tiflow/cdc/model"
+	"github.com/pingcap/tiflow/cdc/owner"
+	"github.com/pingcap/tiflow/cdc/scheduler"
+	"github.com/pingcap/tiflow/cdcv2/metadata"
+	msql "github.com/pingcap/tiflow/cdcv2/metadata/sql"
+	"github.com/pingcap/tiflow/pkg/config"
+	cdcContext "github.com/pingcap/tiflow/pkg/context"
+	cerror "github.com/pingcap/tiflow/pkg/errors"
+	"github.com/pingcap/tiflow/pkg/upstream"
+	"go.uber.org/zap"
+	"gorm.io/gorm"
+)
+
+// Impl implements the owner interface.
+type Impl struct {
```

Comment:
How about rename this to `Owner`? 

I think the interface name should be `Owner`, and the implementation name is `owner`

Reply:
owner is a part of the package name,  can not pass the lint checking

Reply:
done